cli/command/stack/kubernetes/convert.go

```diff
@@ -260,7 +306,20 @@ func fromComposeServiceConfig(s composeTypes.ServiceConfig) latest.ServiceConfig
 		User:            userID,
 		Volumes:         fromComposeServiceVolumeConfig(s.Volumes),
 		WorkingDir:      s.WorkingDir,
+		PullSecret:      pullSecret,
+		PullPolicy:      pullPolicy,
+	}, nil
+}
+
+func resolveServiceExtra(s composeTypes.ServiceConfig, field string) (string, error) {
+	if iface, ok := s.Extras[field]; ok {
+		value, ok := iface.(string)
+		if !ok {
+			return "", errors.Errorf("value %v type is %T, should be a string", iface, iface)
```

Comment:
Add also the field in the error message? It will add more context to resolve the error.